app/build.gradle

```diff
@@ -70,6 +70,9 @@ android {
         archivesBaseName = "${versionName}(${versionCode})"
 
         buildConfigField "String", "INFURA_KEY", "\"set the infura key here\""
+        buildConfigField "String", "LITEWALLET_API_URL_PROD", "\"https://api-prod.lite-wallet.org\""
```

Comment:
thanks @vsima .  We have all the constants in BRConstants. It makes it easy to find.  Is there benefit from changing this?